### PR TITLE
DQL Updated for Proxy Monitoring

### PR DESCRIPTION
**Pull request by rakshitshetty22**
We have Updated the Proxy Monitoring Usecases

### CODE REVIEW

File: workbooks/Advanced Threat Detection/Proxy Monitoring/onedrive_browsing_traffic_profile_ff93ab00d61849eab1a018549edd42a6.yml [hunk 1]
```diff
@@ -1,52 +1,67 @@
...
-    AND $Domain = "onedrive.live.com") AND NOT ($URL LIKE "http%://onedrive.live.com/%"))
+- type: dql
+  query: stream=webfilter where ((method="GET" and url like "%_manifest=wac" and domain="onedrive.live.com")
+    and not (url like "http%://onedrive.live.com/%")) | groupby srcip
 - type: signal
   DetectionName: CobaltStrike Malleable OneDrive Browsing Traffic Profile
   DetectionTactic: Command and Control
   DetectionTechnique: Application Layer Protocol
   DetectionScore: 5
   DetectionConfidence: Low
```
Comment (aman1808) [inline workbooks/Advanced Threat Detection/Proxy Monitoring/onedrive_browsing_traffic_profile_ff93ab00d61849eab1a018549edd42a6.yml:23]: missing detection severity

File: workbooks/Advanced Threat Detection/Proxy Monitoring/solarwinds_backdoor_c2_host_name_detected_8800334c1d054398ae3f8cc46082f899.yml [hunk 2]
```diff
@@ -1,69 +1,74 @@
 id: 8800334c1d054398ae3f8cc46082f899
 name: Solarwinds backdoor C2 host name detected
 path: /Advanced Threat Detection/Proxy Monitoring
-description:  This rule detects the Sunburst C2 channel and any suspicious activity associated with the host name, which is typically used by malicious actors to establish a connection between a compromised system and the attacker's command and control server.
+description: ''
```
Comment (aman1808) [inline workbooks/Advanced Threat Detection/Proxy Monitoring/solarwinds_backdoor_c2_host_name_detected_8800334c1d054398ae3f8cc46082f899.yml:4]: why is the description deleted?

File: workbooks/Advanced Threat Detection/Proxy Monitoring/telegram_api_access_fbf015539fa141d1bbe6c4695e17f152.yml [hunk 3]
```diff
@@ -1,53 +1,71 @@
 id: fbf015539fa141d1bbe6c4695e17f152
 name: Telegram API Access
 path: /Advanced Threat Detection/Proxy Monitoring
-description: Detects suspicious requests to Telegram API without the usual Telegram
-  User-Agent.
+description: ''
```
Comment (aman1808) [inline workbooks/Advanced Threat Detection/Proxy Monitoring/telegram_api_access_fbf015539fa141d1bbe6c4695e17f152.yml:4]: desc deleted?

File: workbooks/Advanced Threat Detection/Proxy Monitoring/turla_comrat_malware_37f1c7de9f744ae2bf2f3397c4190548.yml [hunk 4]
```diff
@@ -1,62 +1,87 @@
 id: 37f1c7de9f744ae2bf2f3397c4190548
 name: Turla ComRAT
 path: /Advanced Threat Detection/Proxy Monitoring
-description: Detects event where ComRAT malware is detected on a system. The malware is typically delivered via spear-phishing emails and is designed to provide the attackers with remote access to the compromised systems.
+description: ''
```
Comment (aman1808) [inline workbooks/Advanced Threat Detection/Proxy Monitoring/turla_comrat_malware_37f1c7de9f744ae2bf2f3397c4190548.yml:4]: desc deleted?

File: workbooks/Advanced Threat Detection/Proxy Monitoring/windows_powershell_user_agent_1b613f156eaf47ac8b9ad17d2ca842ba.yml [hunk 5]
```diff
@@ -1,62 +1,85 @@
 id: 1b613f156eaf47ac8b9ad17d2ca842ba
 name: Windows PowerShell User Agent
 path: /Advanced Threat Detection/Proxy Monitoring
-description: This rule detects windows powerShell web access.
+description: ''
```
Comment (aman1808) [inline workbooks/Advanced Threat Detection/Proxy Monitoring/windows_powershell_user_agent_1b613f156eaf47ac8b9ad17d2ca842ba.yml:4]: add desc

File: workbooks/Advanced Threat Detection/Proxy Monitoring/windows_webdav_useragent_b9614c55a4d145f181ab386c936220f6.yml [hunk 6]
```diff
@@ -1,56 +1,77 @@
 id: b9614c55a4d145f181ab386c936220f6
 name: Windows WebDAV User Agent
 path: /Advanced Threat Detection/Proxy Monitoring
-description: This rule detects the Windows DownloadCradle WebDAV User Agent.
+description: ''
```
Comment (aman1808) [inline workbooks/Advanced Threat Detection/Proxy Monitoring/windows_webdav_useragent_b9614c55a4d145f181ab386c936220f6.yml:4]: add desc

**Review by aman1808 — CHANGES_REQUESTED** [review]

**Comment by aman1808** [comment]
@rakshitshetty22 @hemantvish97 Need to validate the detection score for all the use cases

File: workbooks/Advanced Threat Detection/Proxy Monitoring/empty_user_agent_3b63781a207d4d23b4a714bb31916505.yml [hunk 7]
```diff
@@ -4,47 +4,60 @@ path: /Advanced Threat Detection/Proxy Monitoring
...
 entity_type: workbooks
 blocks:
 - type: text
   code: "|Author|References|\n                 |------|--------|\n               \
     \  |Florian Roth|https://twitter.com/Carlos_Perez/status/883455096645931008|"
-- type: sql
-  query: SELECT $System, $SrcIP, $UserAgent FROM WEBFILTER WHERE $UserAgent = "" GROUP
-    BY $System, $SrcIP, $UserAgent
+- type: dql
+  query: stream=webfilter where useragent="" | groupby system, srcip, useragent
```
[the remark this answers is not in the document]
Comment (rakshitshetty22) [reply workbooks/Advanced Threat Detection/Proxy Monitoring/empty_user_agent_3b63781a207d4d23b4a714bb31916505.yml]: Done with the changes.

File: workbooks/Advanced Threat Detection/Proxy Monitoring/onedrive_browsing_traffic_profile_ff93ab00d61849eab1a018549edd42a6.yml (hunk 1 above)
Comment (rakshitshetty22) [reply workbooks/Advanced Threat Detection/Proxy Monitoring/onedrive_browsing_traffic_profile_ff93ab00d61849eab1a018549edd42a6.yml:24]: Updated the severity.

File: workbooks/Advanced Threat Detection/Proxy Monitoring/solarwinds_backdoor_c2_host_name_detected_8800334c1d054398ae3f8cc46082f899.yml (hunk 2 above)
Comment (rakshitshetty22) [reply workbooks/Advanced Threat Detection/Proxy Monitoring/solarwinds_backdoor_c2_host_name_detected_8800334c1d054398ae3f8cc46082f899.yml]: Updated the Description.

File: workbooks/Advanced Threat Detection/Proxy Monitoring/suspicious_user_agent_e10722c6f4c4485f86b6c318a0b1788c.yml [hunk 8]
```diff
@@ -1,68 +1,91 @@
 id: e10722c6f4c4485f86b6c318a0b1788c
 name: Suspicious User Agent
 path: /Advanced Threat Detection/Proxy Monitoring
-description: Detects suspicious malformed user agent strings in proxy logs
+description: ''
```
[the remark this answers is not in the document]
Comment (rakshitshetty22) [reply workbooks/Advanced Threat Detection/Proxy Monitoring/suspicious_user_agent_e10722c6f4c4485f86b6c318a0b1788c.yml]: Updated the Description.

File: workbooks/Advanced Threat Detection/Proxy Monitoring/turla_comrat_malware_37f1c7de9f744ae2bf2f3397c4190548.yml (hunk 4 above)
Comment (rakshitshetty22) [reply workbooks/Advanced Threat Detection/Proxy Monitoring/turla_comrat_malware_37f1c7de9f744ae2bf2f3397c4190548.yml]: Updated the Description.

File: workbooks/Advanced Threat Detection/Proxy Monitoring/windows_powershell_user_agent_1b613f156eaf47ac8b9ad17d2ca842ba.yml (hunk 5 above)
Comment (rakshitshetty22) [reply workbooks/Advanced Threat Detection/Proxy Monitoring/windows_powershell_user_agent_1b613f156eaf47ac8b9ad17d2ca842ba.yml]: Updated the Description.

File: workbooks/Advanced Threat Detection/Proxy Monitoring/windows_webdav_useragent_b9614c55a4d145f181ab386c936220f6.yml (hunk 6 above)
Comment (rakshitshetty22) [reply workbooks/Advanced Threat Detection/Proxy Monitoring/windows_webdav_useragent_b9614c55a4d145f181ab386c936220f6.yml]: Updated the Description.

**Review by aman1808 — APPROVED** [review]

**Pull request merged**